src/resources/views/fields/number.blade.php

```diff
@@ -10,6 +10,10 @@
         	name="{{ $field['name'] }}"
             id="{{ $field['name'] }}"
             value="{{ old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' )) }}"
+            min="{{ $field['min'] }}"
```

Comment:
What happens if they're not set?

Reply:
I though about it but hadn't time to fix it for the last 2 days, btw thanks for the review and I will fix that